src/com/company/model/DatabasePropertiesProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.company.model;
+
+import com.company.view.ConsoleReader;
+import java.sql.*;
+
+/**
+ * Created by yulia on 02.10.16.
+ */
+public class DatabasePropertiesProvider {
```

Comment:
**PropertiesProvider** is a class, that provides properties stored in files, DB or another storage. But it returns properties that you should input by yourself using console. Pick some better name.